download_models/downloadOVMSModels.sh

```diff
@@ -65,10 +76,16 @@ getOVMSModelFiles() {
 
 if [ $segmentationModelDownloaded -eq 0 ]; then
     echo "download segmentation model..."
-    mkdir -p "$localSegmentationMdlDirName/FP16-INT8/1"
+    mkdir -p "$localPersonVehicleDetectionMdlDirName/FP16-INT8/1"
     getOVMSModelFiles $segmentation $pipelineZooModel$segmentation $modelPrecisionFP16INT8 $localSegmentationMdlDirName
 fi
 
+if [ $personVehicleModelDownloaded -eq 0 ]; then
+    echo "download people vehicle model..."
+    mkdir -p "$localSegmentationMdlDirName/FP16-INT8/1"
```

Comment:
should this be personVehicleModelDirName?  the variable naming is confusing...

Reply:
yes